src/pages/Routes/AuthRoutes.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { useContext } from 'react';
+import Redirect from 'react-router-dom';
+import { AuthContext } from '../../context/authContext';
+
+/**
+ * This component will be used to wrap everything that needs Authentication to access. It uses the
+ * 'AuthContext' to check whether the user is authenticated. If yes, then it will continue to show
+ * the requested components, otherewise the user is redirected to '/' which is the login screen.
+ */
+export default function AuthRoutes({ children }) {
+  const { authenticated } = useContext(AuthContext);
+
+  if (authenticated) {
+    return children;
+  }
+
+  return <Redirect to="/" />;
```

Comment:
do we want to redirect to "/login" instead?

Reply:
Yeah that sounds more logical! Will make that change